packages/components/src/TabBar/TabBar.scss

```diff
@@ -10,13 +10,62 @@
 	background-image: url('data:image/svg+xml;utf8,<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16"><path d="M8 13.6l-8-8L2.6 3 8 8.4 13.4 3 16 5.6z" fill="#{hex-to-url($color)}"/></svg>');
 }
 
+$underline-height: 2px;
+
 .tc-tab-bar {
 	white-space: nowrap;
 	overflow: hidden;
 
 	& &-item {
 		float: none;
 		display: inline-block;
+
+		&-icon {
+			margin-right: $padding-smaller;
+			width: $svg-sm-size;
+			height: $svg-sm-size;
+			font-size: $svg-sm-size;
+		}
+
+		button {
```

Comment:
Seems that need to be in `text-transform: capitalize;` in frontify no? There is a style that need to be override.

Reply:
Can you try not using element-based selectors, but using class instead

Reply:
It's better to use text-transform none here, for example truncation section from fontify has lowercase words
![image](https://user-images.githubusercontent.com/46000544/65419474-64df3d80-de07-11e9-848f-5265fbbda1db.png)

Reply:
Element is generated by NavItem bottstrap component. I tried to apply class for this component, but didn't find the way ho to do it https://react-bootstrap.github.io/components/navs/#nav-item-props

Reply:
Element is generated by NavItem bottstrap component. I tried to apply class for this component, but didn't find the way ho to do it https://react-bootstrap.github.io/components/navs/#nav-item-props. className is applied to li element, not to button. But in my case I need button to select

Reply:
You have to check [the doc for bootstrap 3](https://5c507d49471426000887a6a7--react-bootstrap.netlify.com/components/tabs/#tabs-props-pane) instead

Reply:
I checked it, but didn't find styling prop for NavItem child component. For now button style is already applied (tc-tab-bar-item) but not for its button child component
![image](https://user-images.githubusercontent.com/46000544/65424301-e341dc80-de13-11e9-8c2c-ed26c82f0208.png)

Reply:
Did you just try passing `className` ? :D